### PR TITLE
[global] add Kubernetes 1.30 support

### PR DESCRIPTION
**Pull request by sprait**
## Description

Add Kubernetes 1.30 support.

Updates:
```
ccm:

openstack v1.29.0 -> v1.30.0
aws       v1.29.1 -> v1.30.1
vsphere   v1.29.0 -> v1.30.1
azure     v1.29.0 -> v1.30.2
gcp       ccm/v29.0.0 -> ccm/v29.0.1
```
```
csi:

openstack     v1.29.0  -> v1.30.0
vsphere       v2.5.4 -> v3.0.3
provisioner   v4.0.0 -> v5.0.0
attacher      v4.5.0 -> v4.6.1
resizer       v1.9.3 -> v1.11.0
registrar     v2.10.0 -> v2.11.0
snapshotter   v6.3.3 -> v8.0.0
livenessprobe v2.12.0 -> v2.13.0
```

## Why do we need it, and what problem does it solve?

We want it.

## What is the expected result?

```
root@dev2-master-0:~# kubectl  version
Client Version: v1.30.1
Kustomize Version: v5.0.4-0.20230601165947-6ce0bf390ce3
Server Version: v1.30.1
root@dev2-master-0:~# kubectl  get node -o wide
NAME                               STATUS   ROLES                  AGE    VERSION   INTERNAL-IP       EXTERNAL-IP     OS-IMAGE             KERNEL-VERSION      CONTAINER-RUNTIME
dev2-master-0                      Ready    control-plane,master   167m   v1.30.1   192.168.199.71    95.217.82.143   Ubuntu 22.04.1 LTS   5.15.0-46-generic   containerd://1.7.13
dev2-worker-f8f362c5-5fc4c-bmd26   Ready    worker                 155m   v1.30.1   192.168.199.236   <none>          Ubuntu 22.04.1 LTS   5.15.0-46-generic   containerd://1.7.13
```

## Checklist
- [ ] The code is covered by unit tests.
- [x] e2e tests passed.
- [ ] Documentation updated according to the changes.
- [x] Changes were tested in the Kubernetes cluster manually.

## Changelog entries

```changes
section: global
type: feature
summary: Add Kubernetes 1.30 support.
impact: All control plane components will restart.
impact_level: default
---
section: global
type: feature
summary: Remove support Kubernetes 1.25.
impact: The minimum supported Kubernetes version is 1.26.
impact_level: default
---
section: cloud-provider-aws
type: fix
summary: Fix patch application for Kubernetes 1.29.
impact_level: low
---
section: kube-rbac-proxy
type: fix
summary: Fix patch application for all Kubernetes versions.
impact: Components that use _kube-rbac-proxy_ will restart.
impact_level: default
---
section: cloud-provider-vsphere
type: feature
summary: Update CSI version.
impact_level: default
---
section: cloud-provider-vsphere
type: feature
summary: Disable root reservation for PVC.
impact_level: default
```

### CODE REVIEW

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @sprait. I've started the workflow for label 'e2e/run/aws'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @sprait. I've started the workflow for label 'e2e/run/eks'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @sprait. I've started the workflow for label 'e2e/run/yandex-cloud'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @sprait. I've started the workflow for label 'e2e/run/static'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @sprait. I've started the workflow for label 'e2e/run/azure'...

**Comment by sprait** [comment]
/e2e/abort static;Static;containerd;1.30 9395747612 9395747612-1-con-1-30 /sys/deckhouse-oss/install:pr8525 astra@192.168.199.39

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @sprait. I've recognized your '/e2e/abort' command and started the workflow...

**Comment by sprait** [comment]
/e2e/abort azure;Standard;containerd;1.30 9395748094 9395748094-1-con-1-30 /sys/deckhouse-oss/install:pr8525 azureuser@4.180.18.80

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @sprait. I've recognized your '/e2e/abort' command and started the workflow...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @sprait. I've started the workflow for label 'e2e/run/static'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @sprait. I've started the workflow for label 'e2e/run/azure'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @sprait. I've started the workflow for label 'e2e/run/gcp'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @sprait. I've started the workflow for label 'e2e/run/openstack'...

**Comment by sprait** [comment]
/e2e/abort openstack;Standard;containerd;1.30 9398497367 9398497367-1-con-1-30 /sys/deckhouse-oss/install:pr8525 redos@45.12.230.160

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @sprait. I've recognized your '/e2e/abort' command and started the workflow...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @sprait. I've started the workflow for label 'e2e/run/openstack'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @sprait. I've started the workflow for label 'e2e/run/openstack'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @sprait. I've started the workflow for label 'e2e/run/aws'...

**Comment by sprait** [comment]
/e2e/abort aws;WithoutNAT;containerd;1.30 9449889985 9449889985-1-con-1-30 /sys/deckhouse-oss/install:pr8525 centos@18.199.201.67

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @sprait. I've recognized your '/e2e/abort' command and started the workflow...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @sprait. I've started the workflow for label 'e2e/run/openstack'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @sprait. I've started the workflow for label 'e2e/run/openstack'...

File: ee/candi/version_map.yml [hunk 1]
```diff
@@ -23,3 +23,8 @@ k8s:
       vcd: v1.2.0
     ccm:
       vcd: 1.4.1
+  '1.30':
+    clusterApi:
+      vcd: v1.3.0
+    ccm:
+      vcd: 1.6.0
```
Comment (RomanenkoDenys) [inline ee/candi/version_map.yml:30]: We cannot simply change versions of vcd CCM and CSI

File: ee/modules/030-cloud-provider-vcd/images/capcd-controller-manager/werf.inc.yaml [hunk 2]
```diff
@@ -31,7 +31,7 @@ shell:
     - export GOPROXY={{ $.GOPROXY }}
     - git clone --depth 1 --branch {{ $value.clusterApi.vcd }} {{ $.SOURCE_REPO }}/vmware/cluster-api-provider-cloud-director /src
     - cd /src
-    - test -d /patches/ && for patchfile in /patches/*.patch ; do git apply ${patchfile} ; done
+    - find /patches/ -name '*.patch' | xargs git apply --verbose
```
Comment (RomanenkoDenys) [inline ee/modules/030-cloud-provider-vcd/images/capcd-controller-manager/werf.inc.yaml:34]:
```suggestion
    - git apply /patches/*.patch --verbose
```

File: ee/modules/030-cloud-provider-vcd/images/cloud-controller-manager/werf.inc.yaml [hunk 3]
```diff
@@ -35,7 +35,7 @@ shell:
   - mkdir -p /src
   - git clone --depth 1 --branch {{ $value.ccm.vcd }} {{ $.SOURCE_REPO }}/vmware/cloud-provider-for-cloud-director.git /src
   - cd /src
-  - test -d /patches/{{ $version }} && for patchfile in /patches/{{ $version }}/*.patch ; do git apply ${patchfile} ; done
+  - find /patches/{{ $version }} -name '*.patch' | sort | xargs git apply --verbose
```
Comment (RomanenkoDenys) [inline ee/modules/030-cloud-provider-vcd/images/cloud-controller-manager/werf.inc.yaml:38]:
```suggestion
  - git apply /patches/{{ $version }}/*.patch --verbose
```

File: ee/modules/030-cloud-provider-vsphere/images/cloud-controller-manager/werf.inc.yaml [hunk 4]
```diff
@@ -36,7 +38,7 @@ shell:
   - git clone https://github.com/kubernetes/cloud-provider-vsphere.git /src
   - cd /src
   - git checkout {{ $value.ccm.vsphere }}
-  - test -d /patches/{{ $version }} && for patchfile in /patches/{{ $version }}/*.patch ; do patch -p1 < ${patchfile} ; done
+  - find /patches/{{ $version }} -name '*.patch' | sort | xargs git apply --verbose
```
Comment (RomanenkoDenys) [inline ee/modules/030-cloud-provider-vsphere/images/cloud-controller-manager/werf.inc.yaml:41]:
```suggestion
  - git apply /patches/{{ $version }}/*.patch --verbose
```

File: modules/000-common/images/kube-rbac-proxy/werf.inc.yaml [hunk 5]
```diff
@@ -26,7 +26,7 @@ shell:
   - git clone --depth 1 --branch v0.11.0 {{ .SOURCE_REPO }}/brancz/kube-rbac-proxy.git /src
   install:
   - cd /src
-  - test -d /patches && for patchfile in /patches/*.patch ; do patch -p1 < ${patchfile}; done
+  - find /patches -name '*.patch' | sort | xargs git apply --verbose
```
Comment (RomanenkoDenys) [inline modules/000-common/images/kube-rbac-proxy/werf.inc.yaml:29]:
```suggestion
  - git apply /patches/*.patch --verbose
```

File: modules/030-cloud-provider-aws/images/cloud-controller-manager/werf.inc.yaml [hunk 6]
```diff
@@ -36,7 +38,7 @@ shell:
   - mkdir /src
   - wget https://github.com/kubernetes/cloud-provider-aws/archive/refs/tags/{{ $value.ccm.aws }}.tar.gz -O - | tar -xz --strip-components=1 -C /src/
   - cd /src
-  - test -d /patches/{{ $version }} && for patchfile in /patches/{{ $version }}/*.patch ; do patch -p1 < ${patchfile}; done
+  - find /patches/{{ $version }} -name '*.patch' | sort | xargs git apply --verbose
```
Comment (RomanenkoDenys) [inline modules/030-cloud-provider-aws/images/cloud-controller-manager/werf.inc.yaml:41]:
```suggestion
  - git apply /patches/{{ $version }}/*.patch --verbose
```

File: modules/040-control-plane-manager/images/control-plane-manager/werf.inc.yaml [hunk 7]
```diff
@@ -22,7 +22,7 @@ docker:
 {{- end }}
 ---
 artifact: {{ $.ModuleName }}/controller-artifact
-from: {{ $.Images.BASE_GOLANG_21_ALPINE }}
+from: {{ $.Images.BASE_GOLANG_22_ALPINE }}
```
Comment (RomanenkoDenys) [inline modules/040-control-plane-manager/images/control-plane-manager/werf.inc.yaml:25]: Please use _DEV image version and install packages is not needed

File: modules/040-node-manager/images/cluster-autoscaler/werf.inc.yaml [hunk 8]
```diff
@@ -25,12 +27,15 @@ shell:
   install:
   - mkdir /src
   - cd /src
-{{- if semverCompare ">=1.28" $version }}
+{{- if semverCompare ">=1.30" $version }}
+  - git clone --depth 1 --branch v1.28.{{ $value.clusterAutoscalerPatch }} {{ $.SOURCE_REPO }}/gardener/autoscaler.git .
+  - find /patches/{{ $version }} -name '*.patch' | sort | xargs git apply --verbose
```
Comment (RomanenkoDenys) [inline modules/040-node-manager/images/cluster-autoscaler/werf.inc.yaml:32]:
```suggestion
  - git apply /patches/{{ $version }}/*.patch --verbose
```

File: modules/040-node-manager/images/cluster-autoscaler/werf.inc.yaml [hunk 9]
```diff
@@ -25,12 +27,15 @@ shell:
...
+{{- else if semverCompare ">=1.28" $version }}
   - git clone --depth 1 --branch v1.27.{{ $value.clusterAutoscalerPatch }} {{ $.SOURCE_REPO }}/gardener/autoscaler.git .
-  - test -d /patches/1.27 && for patchfile in /patches/1.27/*.patch ; do patch -p1 < ${patchfile}; done
+  - find /patches/1.27 -name '*.patch' | xargs git apply --verbose
```
Comment (RomanenkoDenys) [inline modules/040-node-manager/images/cluster-autoscaler/werf.inc.yaml:35]:
```suggestion
  - git apply /patches/1.27/*.patch --verbose
```

File: modules/040-node-manager/images/cluster-autoscaler/werf.inc.yaml [hunk 10]
```diff
@@ -25,12 +27,15 @@ shell:
...
 {{- else }}
   - git clone --depth 1 --branch v{{ $version }}.{{ $value.clusterAutoscalerPatch }} {{ $.SOURCE_REPO }}/gardener/autoscaler.git .
-  - test -d /patches/{{ $version }} && for patchfile in /patches/{{ $version }}/*.patch ; do patch -p1 < ${patchfile}; done
+  - find /patches/{{ $version }} -name '*.patch' | sort | xargs git apply --verbose
```
Comment (RomanenkoDenys) [inline modules/040-node-manager/images/cluster-autoscaler/werf.inc.yaml:38]:
```suggestion
  - git apply /patches/{{ $version }}/*.patch --verbose
```

File: modules/101-cert-manager/images/cert-manager-controller/werf.inc.yaml [hunk 11]
```diff
@@ -29,6 +29,6 @@ shell:
   - cd /build
   - git clone -b "v{{ $version }}" --single-branch --depth=1 {{ $.SOURCE_REPO }}/jetstack/cert-manager.git
   - cd /build/cert-manager
-  - test -d /patches && for patchfile in /patches/*.patch ; do patch -p1 < ${patchfile}; done
+  - find /patches -name '*.patch' | xargs git apply --verbose
```
Comment (RomanenkoDenys) [inline modules/101-cert-manager/images/cert-manager-controller/werf.inc.yaml:32]:
```suggestion
  - git apply /patches/*.patch --verbose
```

File: modules/200-operator-prometheus/images/prometheus-operator/werf.inc.yaml [hunk 12]
```diff
@@ -30,7 +30,7 @@ shell:
     - cd /prometheus-operator
     - export VERSION=$(cat VERSION | tr -d " \t\n\r")
     - export GO_BUILD_LDFLAGS="-X github.com/prometheus/common/version.Version=${VERSION} -X github.com/prometheus/common/version.BuildUser=deckhouse"
-    - test -d /patches && for patchfile in /patches/*.patch ; do patch -p1 < ${patchfile}; done
+    - find /patches -name '*.patch' | xargs git apply --verbose
```
Comment (RomanenkoDenys) [inline modules/200-operator-prometheus/images/prometheus-operator/werf.inc.yaml:33]:
```suggestion
    - git apply /patches/*.patch --verbose
```

**Review by RomanenkoDenys — CHANGES_REQUESTED** [review]
some nits

File: deckhouse-controller/files/bashrc [hunk 13]
```diff
@@ -2,5 +2,21 @@ PS1='\[\033[01;30m\][deckhouse]\[\033[00m\] \[\033[01;33m\]\u@\h\[\033[01;34m\]
 
 source /etc/bashrc.d/bash_completion.sh
 
-eval "$(kubectl-1.28 completion bash)"
+
+if [ -s /tmp/kubectl_version ]; then
+ kubernetes_version="$(cat /tmp/kubectl_version)"
+else
+ kubectl_version="1.27"
+fi
```
[the remark this answers is not in the document]
Comment (sprait) [reply deckhouse-controller/files/bashrc:10]: it is file created by `deckhouse/global-hooks/discovery/kubernetes_version.go`, it contains the min current version of k8s in the cluster.

File: ee/candi/version_map.yml (hunk 1 above)
Comment (sprait) [reply ee/candi/version_map.yml]: I had rechecked supported version and returned the version already in use. 
Next version clusterApi requirement 37.2 VMware Cloud Director API

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @sprait. I've started the workflow for label 'e2e/run/yandex-cloud'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @sprait. I've started the workflow for label 'e2e/run/aws'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @sprait. I've started the workflow for label 'e2e/run/static'...

**Comment by sprait** [comment]
/e2e/abort static;Static;containerd;1.30 9602718778 9602718778-1-con-1-30 /sys/deckhouse-oss/install:pr8525

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @sprait. I've recognized your '/e2e/abort' command and started the workflow...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @sprait. I've started the workflow for label 'e2e/run/static'...

**Review by RomanenkoDenys — APPROVED** [review]
LGTM

**Review by name212 — APPROVED** [review]

**Review by z9r5 — APPROVED** [review]

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @sprait. I've started the workflow for label 'e2e/run/vsphere'...

**Comment by sprait** [comment]
/e2e/abort vsphere;Standard;containerd;1.30 9657780874 9657780874-1-con-1-30 /sys/deckhouse-oss/install:pr8525

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @sprait. I've recognized your '/e2e/abort' command and started the workflow...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @sprait. I've started the workflow for label 'e2e/run/vsphere'...

**Comment by sprait** [comment]
/e2e/abort vsphere;Standard;containerd;1.30 9657993227 9657993227-1-con-1-30 /sys/deckhouse-oss/install:pr8525

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @sprait. I've recognized your '/e2e/abort' command and started the workflow...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @sprait. I've started the workflow for label 'e2e/run/vsphere'...

**Comment by sprait** [comment]
/e2e/abort vsphere;Standard;containerd;1.30 9659931247 9659931247-1-con-1-30 /sys/deckhouse-oss/install:pr8525

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @sprait. I've recognized your '/e2e/abort' command and started the workflow...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @sprait. I've started the workflow for label 'e2e/run/vsphere'...

**Comment by sprait** [comment]
/e2e/abort vsphere;Standard;containerd;1.30 9661314015 9661314015-1-con-1-30 /sys/deckhouse-oss/install:pr8525

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @sprait. I've recognized your '/e2e/abort' command and started the workflow...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @sprait. I've started the workflow for label 'e2e/run/vsphere'...

**Comment by sprait** [comment]
/e2e/abort vsphere;Standard;containerd;1.30 9661680095 9661680095-1-con-1-30 /sys/deckhouse-oss/install:pr8525

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @sprait. I've recognized your '/e2e/abort' command and started the workflow...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @sprait. I've started the workflow for label 'e2e/run/vsphere'...

**Comment by sprait** [comment]
/e2e/abort vsphere;Standard;containerd;1.30 9664591394 9664591394-1-con-1-30 /sys/deckhouse-oss/install:pr8525

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @sprait. I've recognized your '/e2e/abort' command and started the workflow...

**Comment by sprait** [comment]
/e2e/abort vsphere;Standard;containerd;1.30 9665403571 9665403571-1-con-1-30 /sys/deckhouse-oss/install:pr8525 ubuntu@10.220.203.43

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @sprait. I've recognized your '/e2e/abort' command and started the workflow...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @sprait. I've started the workflow for label 'e2e/run/vsphere'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @sprait. I've started the workflow for label 'e2e/run/vsphere'...

**Comment by sprait** [comment]
/e2e/abort vsphere;Standard;containerd;1.27 9666177851 9666177851-1-con-1-27 /sys/deckhouse-oss/install:pr8525 ubuntu@10.220.203.43

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @sprait. I've recognized your '/e2e/abort' command and started the workflow...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @sprait. I've started the workflow for label 'e2e/run/vsphere'...

**Comment by sprait** [comment]
/e2e/abort vsphere;Standard;containerd;1.27 9676001923 9676001923-1-con-1-27 /sys/deckhouse-oss/install:pr8525 ubuntu@10.220.203.71

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @sprait. I've started the workflow for label 'e2e/run/vsphere'...

**Comment by sprait** [comment]
/e2e/abort vsphere;Standard;containerd;1.27 9677512430 9677512430-1-con-1-27 /sys/deckhouse-oss/install:pr8525 ubuntu@10.220.203.136

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @sprait. I've recognized your '/e2e/abort' command and started the workflow...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @sprait. I've started the workflow for label 'e2e/run/openstack'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @sprait. I've started the workflow for label 'e2e/run/vsphere'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @sprait. I've started the workflow for label 'e2e/run/vsphere'...

**Comment by sprait** [comment]
/e2e/abort vsphere;Standard;containerd;1.27 9678194045 9678194045-1-con-1-27 /sys/deckhouse-oss/install:pr8525 ubuntu@10.220.203.148

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @sprait. I've recognized your '/e2e/abort' command and started the workflow...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @sprait. I've started the workflow for label 'e2e/run/vsphere'...

**Comment by sprait** [comment]
/e2e/abort vsphere;Standard;containerd;1.27 9679035694 9679035694-1-con-1-27 /sys/deckhouse-oss/install:pr8525 ubuntu@10.220.203.148

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @sprait. I've recognized your '/e2e/abort' command and started the workflow...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @sprait. I've started the workflow for label 'e2e/run/vsphere'...

**Comment by sprait** [comment]
/e2e/abort vsphere;Standard;containerd;1.27 9679813602 9679813602-1-con-1-27 /sys/deckhouse-oss/install:pr8525 ubuntu@10.220.203.144

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @sprait. I've recognized your '/e2e/abort' command and started the workflow...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @sprait. I've started the workflow for label 'e2e/run/vsphere'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @sprait. I've started the workflow for label 'e2e/run/vsphere'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @sprait. I've started the workflow for label 'e2e/run/vsphere'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @sprait. I've started the workflow for label 'e2e/run/vsphere'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @sprait. I've started the workflow for label 'e2e/run/openstack'...

**Review by vladimirGuryanov — APPROVED** [review]

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @sprait. I've started the workflow for label 'e2e/run/vsphere'...

**Comment by sprait** [comment]
/e2e/abort vsphere;Standard;containerd;1.30 9693557148 9693557148-1-con-1-30 /sys/deckhouse-oss/install:pr8525 ubuntu@10.220.203.43

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @sprait. I've recognized your '/e2e/abort' command and started the workflow...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @sprait. I've started the workflow for label 'e2e/run/vsphere'...

**Comment by sprait** [comment]
/e2e/abort vsphere;Standard;containerd;1.30 9694664326 9694664326-1-con-1-30 /sys/deckhouse-oss/install:pr8525 ubuntu@10.220.203.162

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @sprait. I've recognized your '/e2e/abort' command and started the workflow...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @sprait. I've started the workflow for label 'e2e/run/vsphere'...

**Comment by sprait** [comment]
/e2e/abort vsphere;Standard;containerd;1.30 9699644587 9699644587-1-con-1-30 /sys/deckhouse-oss/install:pr8525 ubuntu@10.220.203.162

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @sprait. I've recognized your '/e2e/abort' command and started the workflow...

**Comment by sprait** [comment]
/e2e/abort vsphere;Standard;containerd;1.30 9702294855 9702294855-1-con-1-30 /sys/deckhouse-oss/install:pr8525 ubuntu@10.220.203.162

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @sprait. I've recognized your '/e2e/abort' command and started the workflow...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @sprait. I've started the workflow for label 'e2e/run/vsphere'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @sprait. I've started the workflow for label 'e2e/run/vsphere'...

**Comment by sprait** [comment]
/e2e/abort vsphere;Standard;containerd;1.30 9712257126 9712257126-1-con-1-30 /sys/deckhouse-oss/install:pr8525 ubuntu@10.220.203.162

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @sprait. I've recognized your '/e2e/abort' command and started the workflow...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @sprait. I've started the workflow for label 'e2e/run/vsphere'...

**Review by RomanenkoDenys — APPROVED** [review]
LGTM

**Review by name212 — APPROVED** [review]

**Pull request merged**